Intersect.Server.Core/Entities/Events/CommandProcessing.cs

```diff
@@ -312,15 +312,30 @@ Stack<CommandInstance> callStack
         )
         {
             var commonEvent = EventBase.Get(command.EventId);
-            if (commonEvent != null)
+            if (commonEvent == null)
             {
-                for (var i = 0; i < commonEvent.Pages.Count; i++)
+                return;
+            }
+
+            for (var i = 0; i < commonEvent.Pages.Count; i++)
+            {
+                if (Conditions.CanSpawnPage(commonEvent.Pages[i], player, instance))
                 {
-                    if (Conditions.CanSpawnPage(commonEvent.Pages[i], player, instance))
-                    {
-                        var commonEventStack = new CommandInstance(commonEvent.Pages[i]);
-                        callStack.Push(commonEventStack);
-                    }
+                    var commonEventStack = new CommandInstance(commonEvent.Pages[i]);
+                    callStack.Push(commonEventStack);
+                }
+            }
+
+            if (player.MapInstanceId == Guid.Empty && !command.OverworldOverride)
+            {
+                return;
+            }
+
+            if (command.AllInInstance)
+            {
+                foreach (var instanceMember in Globals.OnlineList.Where(pl => pl.MapInstanceId == player.MapInstanceId && pl.Id != player.Id).ToArray())
```

Comment:
Can we actually move the Where into just if statements with a continue inside the for loop? This is too long of a line

Reply:
yes, done